pkg/ring/lifecycler.go

```diff
@@ -445,9 +453,22 @@ heartbeatLoop:
 // - add an ingester entry to the ring
 // - copies out our state and tokens if they exist
 func (i *Lifecycler) initRing(ctx context.Context) error {
-	var ringDesc *Desc
+	var (
+		ringDesc       *Desc
+		tokensFromFile Tokens
+		err            error
+	)
+
+	if i.cfg.TokensFilePath != "" {
+		tokensFromFile, err = LoadTokensFromFile(i.cfg.TokensFilePath)
+		if err != nil {
+			level.Error(util.Logger).Log("msg", "error in getting tokens from file", "err", err)
+		}
+	} else {
+		level.Warn(util.Logger).Log("msg", "not loading tokens from file, tokens file path is empty")
```

Comment:
Do we need a warning for this? Maybe debug?

Reply:
Info should be fine as @pstibrany suggested. It would be good to have that info without enabling debug level.